From cd069f80bac56ce05e41ea34b9c5beae4d27abc2 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Wed, 28 Dec 2022 15:45:35 +0800
Subject: [PATCH] pfe: update KERNEL_VERSION to v5.16.0 to fix build error

Currentlly, the S32G kernel branch in linux-yocto is based on v5.10 kernel
of NXP SDK BSP35, so to keep align with SDK, the KERNEL_VERSION(5,15,0) need
update to a bigger KERNEL_VERSION.

Upstream-Status: Pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>

---
 sw/linux-pfeng/pfeng.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sw/linux-pfeng/pfeng.h b/sw/linux-pfeng/pfeng.h
index 9ad82e5..8f91db0 100644
--- a/sw/linux-pfeng/pfeng.h
+++ b/sw/linux-pfeng/pfeng.h
@@ -21,7 +21,7 @@
 #include <linux/kfifo.h>
 #include <linux/mutex.h>
 #include <linux/clk.h>
-#if LINUX_VERSION_CODE < KERNEL_VERSION(5,15,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,16,0)
 #include <linux/pcs/fsl-s32gen1-xpcs.h>
 #define s32cc_phy2xpcs s32gen1_phy2xpcs
 #define s32cc_xpcs_get_ops s32gen1_xpcs_get_ops
@@ -275,7 +275,7 @@ struct pfeng_emac {
 	struct mii_bus			*mii_bus;
 	/* XPCS */
 	struct phy			*serdes_phy;
-#if LINUX_VERSION_CODE < KERNEL_VERSION(5,15,0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,16,0)
 	struct s32gen1_xpcs             *xpcs;
 	const struct s32gen1_xpcs_ops   *xpcs_ops;
 #else
-- 
2.25.1

